src/sphinxcontrib/towncrier/_towncrier.py

```diff
@@ -1,5 +1,5 @@
 """Towncrier related shims."""
```

Comment:
Keep the separator line.
```suggestion
"""Towncrier related shims."""

```